packages/parser/src/schemas/lambda.ts

```diff
@@ -0,0 +1,13 @@
+import { APIGatewayProxyEventV2Schema } from './apigwv2.js';
+
+const LambdaFunctionUrlSchema = APIGatewayProxyEventV2Schema.extend({
+  //  Lambda Function URL follows the API Gateway HTTP APIs Payload Format Version 2.0.
```

Comment:
I'd put this comment above the line like everywhere else, so that it shows up when you hover the `LambdaFunctionUrlSchema` object, i.e. 

```ts
/**
 * text here
 *
*/
const LambdaFunctionUrlSchema ...

```

Reply:
fixed